Doc/Tutorial/chapter_seq_objects.tex

```diff
@@ -13,7 +13,7 @@ \chapter{Sequence objects}
 
 \section{Sequences and Alphabets}
 
-The alphabet object is perhaps the important thing that makes the \verb|Seq| object more than just a string. The currently available alphabets for Biopython are defined in the \verb|Bio.Alphabet| module. We'll use the IUPAC alphabets (\url{http://www.chem.qmw.ac.uk/iupac/}) here to deal with some of our favorite objects: DNA, RNA and Proteins.
+The alphabet object is perhaps the important thing that makes the \verb|Seq| object more than just a string. The currently available alphabets for Biopython are defined in the \verb|Bio.Alphabet| module. We'll use the IUPAC alphabets (\url{https://www.bioinformatics.org/sms/iupac.html}) here to deal with some of our favorite objects: DNA, RNA and Proteins.
```

Comment:
This has moved to http://www.sbcs.qmul.ac.uk/iupac/ (following the Queen Mary redirection as a little guess work).

The page https://www.bioinformatics.org/sms/iupac.html is not an official reference for IUPAC.